wiki/Sprint-4.md

```diff
@@ -0,0 +1,51 @@
+# Sprint 2
+
+## Summary
+
+This Sprint was the first sprint with a two-week duration. This introduced some organizational benefits and drawbacks. While we had less stress to finish something in one week, it was more difficult to estimate the workload over the whole sprint correctly. The two-week sprint interval also removes some overhead and allocates some more time which we can use to work on the project or organize a Retrospective. The latter we did last Friday to improve the work together and check the mood of the group. I think we well progressed this sprint and we are looking forward to implementing some new interesting features.
+
+![Moodboard of the retrospective](image/mood-board.jpg)
+
+
+## Lars Barmettler
+
+The last two weeks I was able to work quite productive. In the first week I could correct a refactoring some ill-favoured code which I wrongly implemented in the previous sprint. This was not that hard to do such that I quickly progressed to my second task. The second task, however, was quite demanding and so I often took consultation by Alexandre. He showed me how to solve this problem in a Kotlin way. I really like that we can ask without hesitation questions each other and getting quickly a respond. We should continue to stick to this mindset for the rest of the project.
+
+## Matthieu Burguburu
+
+My main task for this sprint was to finish the design of the "Prepare game" screen on Figma, and implement both its UI and logic. My time estimates for these tasks were pretty good, but I took some extra unplanned time to implement a feature closely-related to this but which was slightly misplanned. However, this was nothing major and did not hinder me in my main task.
+
+Additionally, I had to finish increasing the coverage on a feature from last sprint. This should not have taken much time, but it lead to some refactoring and was a bit longer than expected.
+
+Overall, I think this 2 week sprint went quite well and did not make it harder to deliver my features than previous shorter sprints.
+
+## Chau Ying Kot (Scrum Master)
```

Comment:
I'm not the scrum master for this sprint ;)

Reply:
oobs sorry